src/controllers/db/statements/getSimilarstatements.tsx

```diff
@@ -13,7 +13,7 @@ export const findSimilarStatements = async (
 		const isProd = false;
 
 		if (isProd) {
-			return prodEndPoint;
+			return localEndPoint;
```

Comment:
Revert these changes and merge the origin/dev to this branch. It was solved there.